docs/usage/configuration.rst

```diff
@@ -1,23 +1,14 @@
----
-nav_order: 6
----
-
-# Configuration
-
+Configuration
+#############
 If you want to use PlaidML from the command line, you can set the
 following environment variables to select the proper configurations for your
 device. This is equivalent to running `plaidml-setup` and selecting these
 settings when prompted.
-
-  * `PLAIDML_EXPERIMENTAL` - (0 or 1) determines whether to enable
-     experimental mode in PlaidML 
-  * `PLAIDML_DEVICE_IDS` - (string) the name of the device to use
-     with PlaidML (to see a list of devices, run `plaidml-setup`)
-
+  * `PLAIDML_EXPERIMENTAL` - (0 or 1) determines whether to enable experimental mode in PlaidML 
+  * `PLAIDML*DEVICE*IDS` - (string) the name of the device to use with PlaidML (to see a list of devices, run ``plaidml\-setup``)
 Below is an example of how to set the device configuration environment variables
 for PlaidML.
+.. code-block::
+  export PLAIDML_EXPERIMENTAL=1
+  export PLAIDML*DEVICE*IDS=opencl*intel*uhd*graphics*630.0
```

Comment:
These are all gone in plaidml-v1. `PLAIDML_DEVICE_IDS` is now `PLAIDML_DEVICE` and a new env var is introduced: `PLAIDML_TARGET`